frontend/src/vault/keysign/shared/KeysignTxOverviewRow.tsx

```diff
@@ -0,0 +1,22 @@
+import { HStack, VStack } from '../../../lib/ui/layout/Stack';
+import { Text } from '../../../lib/ui/text';
+
+export const KeysignTxOverviewRow = ({
+  label,
+  value,
+}: {
+  label: string;
+  value: string;
+}) =>
+  value ? (
```

Comment:
I'm not sure, but I think we have `TxOverviewRow` component that might be doing the same.